calc/index.html

```diff
@@ -32,7 +32,10 @@
 }
 
 function CalcVmem() {
-	var GPDBOtherMem = (parseInt($('#OtherMemory').val())) ? parseInt($('#OtherMemory').val()) : 0;
+    var GPDBOtherMem = (parseInt($('#OtherMemory').val())) ? parseInt($('#OtherMemory').val()) : 0;
+    var GPTextHeapSize = (parseInt($('#GPTextHeapSize').val())) ? parseInt($('#GPTextHeapSize').val()) : 0;
+    var GPtextNumOfPids =  (parseInt($('#GPTextNodes').val())) ? parseInt($('#GPTextNodes').val()) : 0;
+    GPDBOtherMem += GPTextHeapSize * GPtextNumOfPids;
```

Comment:
Supernitpick: the variable `GPtextNumOfPids` should have a capital "T" in GPText to be consistent (`GPTextNumOfPids`).